docs/user-guide/monitoring.md

```diff
@@ -1,5 +1,9 @@
 # Prometheus and Grafana installation
+Two different methods to install and configure Prometheus and Grafana are described in this doc. 
+- Prometheus and Grafana installation using Pod Annotations. This installs Prometheus and Grafana in the same namespace as NGINX Ingress
+- Prometheus and Grafana installation using Service Monitors. This is more suited for existing Prometheus and Grafana installations that can be configured with NGINX Ingress for scraping the metrics. 
 
+# Prometheus and Grafana installation using Pod Annotations
```

Comment:
This is preferred as second level in markdown so 2 hash symbols may be better

Reply:
Done. I also increased the number of hash symbols for the subheadings under second level heading: `Prometheus and Grafana installation using Pod Annotations` and added bullet points to match the original document format